elide-async/src/main/java/com/yahoo/elide/async/models/AsyncQueryResult.java

```diff
@@ -5,46 +5,33 @@
  */
 package com.yahoo.elide.async.models;
 
-import com.yahoo.elide.annotation.CreatePermission;
-import com.yahoo.elide.annotation.DeletePermission;
-import com.yahoo.elide.annotation.Exclude;
-import com.yahoo.elide.annotation.Include;
-import com.yahoo.elide.annotation.ReadPermission;
-import com.yahoo.elide.annotation.UpdatePermission;
-
 import lombok.Data;
 
+import java.util.Date;
+
 import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.OneToOne;
+import javax.persistence.Embeddable;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
 
 /**
  * Model for Async Query Result.
  */
-@Entity
-@Include(type = "asyncQueryResult")
-@ReadPermission(expression = "Principal is Owner")
-@UpdatePermission(expression = "Prefab.Role.None")
-@CreatePermission(expression = "Prefab.Role.None")
-@DeletePermission(expression = "Prefab.Role.None")
+@Embeddable
 @Data
-public class AsyncQueryResult extends AsyncBase implements PrincipalOwned {
-    @Id
-    @Column(columnDefinition = "varchar(36)")
-    private String id; //Matches id in query.
+
+public class AsyncQueryResult {
 
     private Integer contentLength;
 
     private String responseBody; //success or errors
```

Comment:
In the specs, we wanted to rename this to result. But with embedded can we have this called result as there is another column called result in AsynQuery.

Reply:
Comment for the column should be said Response Body if we rename the column.

Reply:
changed the comment. but do we really want to rename it to result?